docs/input/docs/ci-systems/appveyor.md

```diff
@@ -0,0 +1,37 @@
+---
+Order: 20
+Title: AppVeyor
+Description: Running on AppVeyor
+---
+
+## specifics
+
+None.
+
+## example config
+
+```yaml
+image:
+  - Visual Studio 2019
+
+pull_requests:
+  do_not_increment_build_number: true
+
+skip_branch_with_pr: true
```

Comment:
As a general example, this line should probably not be here.
Or at the very least, it should be commented out.

Reply:
Agreed. 

Also, (thinking about it..) the provided examples should do roughly the same for all systems. 

Something like 
- build on one os (windows?)
- only minimal "triger" configuration 
- caching ? 
- building
- additional steps (like adding .net core and uploading artefacts in gh-actions)

Reply:
> build on one os (windows?)

I think for providers that can run multiple os in parallel (by default), I think it would be fine to have different os for those providers (more to encourage maintainers to test on different platforms than anything else).

The rest of what you mentioned I agree with.

Reply:
I "minimalized" the examples and made them "comparable" (I.e. they all now do the same...)